content/en/quickstart/create-device.md

```diff
@@ -11,3 +11,264 @@ menu:
+3. Start up your Raspberry Pi as described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-setting-up/4)
+4. Configuring your Raspberry Pi described [here](https://projects.raspberrypi.org/en/projects/raspberry-pi-using/9)
+    
+    #### Configure SSH(Remote Access)
+    You make sure to enable SSH to allow remote access to your Raspberry Pi from another computer using SSH. 
+    
+    Simple way to enable SSH as described [here](https://www.raspberrypi.org/documentation/remote-access/ssh/README.md) is to select `Enable` next to `SSH` in the interface tap of `Raspberry Pi configuration` from the `Preferences` menu.
+    
+    ![Interfaces] (https://projects-static.raspberrypi.org/projects/raspberry-pi-using/e3d1233ee773b4f80821995131420119ddb7e40c/en/images/pi-configuration-interfaces.png)
+    
+    Or 
+    
+    You can also configure SSH via `raspi-config` on the Pi terminal. 
+    ```shell script
+    sudo raspi-config
+   
+    # Select 'Interfacing Options'
+    # Navigate to and select 'SSH'
+    # Choose 'Yes'
+    # Select 'Ok'
+    # Choose 'Finish'
+    ```
+
+## Connect to the Raspberry Pi via SSH described [here](https://www.raspberrypi.org/documentation/remote-access/ssh/unix.md)
+1. Open a terminal on your computer.  
+2. Get IP address of the Raspberry Pi board by following command on the terminal. 
+
+    ```
+    ping raspberrypi.local 
+    ```
+    If the ping command doesn't reach raspberrypi.local due to DNS issues, you can connect a keyboard and monitor to the Raspberry Pi and run `ifconfig` command in the terminal.
+
+3. Connect to the Pi via SSH using these commands.
+
+    For Linux or macOS, use either of these commands:
+    ```shell script
+    # replace with current hostname if changed.
+    ssh pi@raspberrypi.local  
+    ```
+    Or 
+    ```shell script
+    # use IP address from previous step
+    ssh pi@ipaddress  
+    ```
+4. Type the password when prompted. Default login username is `pi` and password is `raspberry`.
+
+## Install essential build tools
+1. Update & upgrade all the installed packages.
+    ```shell script
+    sudo apt-get update
+    sudo apt-get upgrade
+    ```
+2. Install build tools and so on.
+    ```shell script
+    sudo apt-get install build-essential git curl openssl vim jq tac awk
+    ```
+
+## Install cloud server (OCF cloud enabled device) 
+1. Download IoTivity-Lite open soure from the GitHub. 
+
+    Continue on your personal PC, from the SSH prompt, go to the home directory and clone the latest open source of IoTivity-Lite.
+    
+    ```shell script
+    git clone https://gitlab.iotivity.org/iotivity/iotivity-lite.git
+    ```
+    Enter in the iotivity-lite folder you just cloned.
+    ```shell script
+    cd iotivity-lite
+    ls 
+    Architecture.png	apps			patches			tools
+    LICENSE.md		deps			port			util
+    Porting.png		include			security
+    README.rst		messaging		swig
+    api			onboarding_tool		tests
+    ```
+
+2. Get cloud configuration.
```

Comment:
This can be completely skipped from the quickstart. And it doesn't belong to this section. This section is only about making iotivity-lite cloud server running on the raspi. You can continue directly with lines 175+. 
This one doc shall contain what you have, how to start it, and how to start cloud_server everytime raspi restarts.

Next markdown will contain documentation how to discover device using mobile app, or optionally go-sdk.

Reply:
First of all, I pushed new commit removed sections to prepare on-boarding, provisioning by obt. How about this new commit (https://github.com/plgd-dev/plgd.dev/pull/1/commits/7b5875c2ebc6e05ff74010dd9179658f42dec076) ?

Reply:
@ondrejtomcik 
Are you willing to not offer onboarding & provisioning by on-boarding tool of iotivity-lite ?

Reply:
I will review the change soon.
No, I don't want to document iotivity-lite onboarding tool. We have our own one, built on top of plgd-dev/sdk , which is also used in the mobile app.